src/components/projects/Entry.jsx

```diff
@@ -68,112 +76,195 @@ function Entry() {
-          name="textInvitation"
-          onBlur={formik.handleBlur}
-          onChange={formik.handleChange}
-          rows={2}
-          value={formik.values.textInvitation}
-          variant="outlined"
-        />
-      </div>
-
-      <MemberList memberList={memberList} setMemberList={setMemberList} />
+  const useStyles = makeStyles({
+    button: {
+      margin: "10px",
+    },
+    customGrayFont: {
+      color: "#d2d2d2",
+      fontSize: "1.2rem",
+      fontWeight: "bold",
+    },
+    subTitle: {
+      fontSize: "1.5rem",
+      fontWeight: "bold",
+    },
+    textInput: {
+      width: "90%",
+    },
+    title: {
+      color: "#DF8930",
+      fontSize: "2.8rem",
+      fontWeight: "bold",
+      margin: "30px",
+    },
+  });
+  const classes = useStyles();
 
-      <Button
+  return (
+    <Container className="container">
+      <Link className="back-button" to="/projects">
+        <ArrowBackIos></ArrowBackIos>
+        Back
+      </Link>
+      <Typography
+        className={classes.title}
         color="primary"
-        data-testid="btn-form"
-        type="submit"
-        variant="contained"
+        gutterBottom
+        variant="h3"
       >
-        {idProject ? "Update" : "Create"}
-      </Button>
-    </form>
+        Let's showcase your project!
+      </Typography>
+      <form
+        autoComplete="off"
+        className="container-form"
+        noValidate
+        onSubmit={formik.handleSubmit}
+      >
+        <Box display="flex" justifyContent="space-between" mb={6}>
+          <Paper className="paper" elevation={3}>
+            <Typography
+              align="justify"
+              className={classes.subTitle}
+              color="primary"
+              gutterBottom
+              variant="h5"
+            >
+              Project information
+            </Typography>
+            <div className="u-mb-1">
+              <Typography align="justify" gutterBottom variant="body1">
+                Title
+              </Typography>
+              <TextField
+                className={classes.textInput}
+                data-testid="input-field"
+                error={hasErrorName}
+                helperText={hasErrorName ? formik.errors.name : ""}
+                id="name"
+                label="Name"
+                name="name"
+                onBlur={formik.handleBlur}
+                onChange={formik.handleChange}
+                type="text"
+                value={formik.values.name}
+                variant="outlined"
+              />
+            </div>
+            <div className="u-mb-1">
+              <Typography align="justify" gutterBottom variant="body1">
+                Owner Contact
+              </Typography>
+              <TextField
+                InputProps={{
+                  readOnly: true,
+                }}
+                className={classes.textInput}
+                data-testid="input-field"
+                helperText="This is the owner contact"
```

Comment:
Remove helperText, it's already specified in the label

Reply:
done